agent/04_agent_configure.sh

```diff
@@ -522,6 +504,10 @@ if [ ! -z "${MIRROR_IMAGES}" ]; then
 
 fi
 
+if [ "${OC_MIRROR}" == "true " && "${MIRROR_MCE}" == "true " ]; then
```

Comment:
https://www.shellcheck.net/wiki/SC2107

Since we are using bash, either

```suggestion
if [[ "${OC_MIRROR}" == "true " && "${MIRROR_MCE}" == "true " ]]; then
```

or

```suggestion
if [ "${OC_MIRROR}" == "true " ] && [ "${MIRROR_MCE}" == "true " ]; then
```